sdk/nodejs/cmd/run/run.ts

```diff
@@ -53,14 +64,38 @@ function projectRootFromProgramPath(program: string): string {
     }
 }
 
+async function npmPackageRootFromProgramPath(programPath: string): Promise<string> {
```

Comment:
I appreciate the surrounding comments. Would you mind adding a comment above this function about what programPath is?